src/Azure/AzureAD/Authentication.AzureADB2C.UI/src/Microsoft.AspNetCore.Authentication.AzureADB2C.UI.csproj

```diff
@@ -6,8 +6,8 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <PackageTags>aspnetcore;authentication;AzureADB2C</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <ProvideApplicationPartFactoryAttributeTypeName>Microsoft.AspNetCore.Mvc.ApplicationParts.NullApplicationPartFactory, Microsoft.AspNetCore.Mvc.Core</ProvideApplicationPartFactoryAttributeTypeName>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <NoWarn>RS0016</NoWarn>
```

Comment:
This (and similar instances) will be removed once we bring in the SDK containing https://github.com/dotnet/sdk/pull/16419 is produced.

Reply:
Mostly curious: What new types and members exist as a result of this change❔

Reply:
We did some work in preview3 to improve the compilation together. Instead of a two-step compilation that produced an assembly containing all the views and the app's assembly, everything is compiled into one (see issue above for more info).

Now that all the compiled views are in one assembly alongside the users app code, we changed it so that the generated view type is compiled as `internal sealed` instead of `public` by default. (see https://github.com/dotnet/aspnetcore/pull/31008). This is the change that I'm waiting to flow through.

Without it, the generated C# produced by the Razor source generator participates in the compilation step for the build. This generated code contains public types that are then subject to the public API analyzer. Back when we were using a two-step compilation to compute the views DLL, we had a separate `CoreCompile` task that ran without this analyzer enabled.

Reply:
This is fixed now!